torchgeo/datamodules/usavars.py

```diff
@@ -24,7 +23,6 @@ def __init__(
         self,
         root_dir: str,
         labels: Sequence[str] = USAVars.ALL_LABELS,
-        transforms: Optional[Callable[[Dict[str, Tensor]], Dict[str, Tensor]]] = None,
```

Comment:
We don't allow transforms to be passed in to any other DataModules

Reply:
This is what we call a slippery slope: https://github.com/microsoft/torchgeo/pull/498#discussion_r843194575

Also see https://github.com/microsoft/torchgeo/discussions/596

Reply:
Haha yeah I know; I can't make up my mind here. If we have this `Optional[Callable]` as an argument, then we can't (easily) initialize a class instance from a configuration file. This will make general-purpose scripts like `predict.py` more difficult to write. On the other hand, our datamodules aren't very useful for experimentation as-is, because good train-time augmentations are a very important.

Reply:
Can't we have a set of default transforms associated with each datamodule? Then users can override them in Python if they need to via this optional callable (experimentation) but we'll still have config files without customization that default to the optimal known transforms (reproduction).

Reply:
Yeah I think that's the best of both worlds. I don't think an "optimal" known transform exists across these datasets -- but a sane default implemented consistently would be a good step. Further, as long as these are optional train transforms, instantiating the DataModule without them shouldn't have any effect on using the DataModule for pre-processing purposes.